File: Day-One/Level-Two/python/level_two.py
def get_duplicate(array):
    new_list = []
    duplicate_list = []
    for number in array:
        if number in new_list:
            if number not in duplicate_list:
                duplicate_list.append(number)
        else:
            new_list.append(number)

    return duplicate_list 


def sort_array(array):
    
    size = len(array)
    for count in range(0, size, 1):

        for index in range(count + 1, size, 1):

            if array[index] >  array[count]:
                array[count], array[index] = array[index], array[count]

    return array
                

File: Day-One/Level-Two/Javascript/level_two.js
function get_duplicate(items){

    let duplicate_list = []
    let seen_list = []
    for(let number of items){
        if( seen_list.includes(number) ){
            if(!duplicate_list.includes(number)) duplicate_list.push(number);
        }

        else{
            seen_list.push(number)
        }
    }

    return duplicate_list
}

module.exports = { get_duplicate }
 
function push_zero(items){

    let size = items.length
    let temp = []
    let tempCount = 0
    
    for(let count = 0; count < size; count++){
        if(items[count] != 0 ){
            temp[tempCount] = items[count]
            tempCount++
        }
    }

    while(tempCount < size){
        temp[tempCount++] = 0
    }

    for(let count = 0; count < size; count++){
        items[count] = temp[count]
    }
    return items
}

module.exports = { push_zero, get_duplicate}
